Sources/LiveViewNative/Modifiers/Text Input and Output Modifiers/TruncationModeModifier.swift

```diff
@@ -0,0 +1,56 @@
+//
+//  TruncationModeModifier.swift
+//  LiveViewNative
+//
+//  Created by Dylan.Ginsburg on 4/18/23.
+//
+
+import SwiftUI
+
+/// Sets the truncation mode for lines of text that are too long to fit in the available space.
+///
+/// ```html
+/// <Text
+///     modifiers={
+///         truncationMode(@native, mode: :tail)
+///     }
+/// >
+///   This text is truncated at the end if it overflows
+/// </Text>
+/// ```
+///
+/// ## Arguments
+/// * ``mode``
+#if swift(>=5.8)
+@_documentation(visibility: public)
+#endif
+struct TruncationModeModifier: ViewModifier, Decodable, Equatable {
+    /// One of the `Text.TruncationMode` enumerations.
```

Comment:
Can you list the possible values here?
```suggestion
    /// One of the `Text.TruncationMode` enumerations.
    ///
    /// Possible values:
    /// * `head`
    /// * `middle`
    /// * `tail`
```